pkg/cache/types/types.go

```diff
@@ -35,8 +35,8 @@ type ResponseType int
 const (
 	Endpoint ResponseType = iota
 	Cluster
-	Route
 	ScopedRoute
+	Route
```

Comment:
this is nitpicky but could this technically be a breaking change? in case someone had comparisons on this type, seems unfortunate but maybe

Reply:
Ah yea that's a good point... hm

Reply:
I think the solution to this imo is just bump the released version. We can note this in the changelog but it's a critical part of the ordering problem.